src/locales/en-US/commands.json

```
@@ -497,5 +497,17 @@
   "aww": {
     "title": "Here is some eyebleach for ya!",
     "commandDescription": "Random image from /r/aww"
+  },
+  "mcquery": {
+    "commandDescription": "Fetch info from a MC:PC server.",
+    "commandUsage": "<address:port>",

```

The port isn't required:
```suggestion
    "commandUsage": "<address>[:port]",
```